superset/models/core.py

```diff
@@ -665,12 +665,13 @@ def export_dashboards(cls, dashboard_ids):
             )
             make_transient(copied_dashboard)
             for slc in copied_dashboard.slices:
+                make_transient(slc)
```

Comment:
This looked like a pretty bad bug, exporting dashboards would mutate slices in odd ways.